src/chart/view.ts

```diff
@@ -1085,3 +1085,5 @@ export const registerGeometry = (name: string, Ctor: any) => {
     return geometry;
   };
 };
+
+export default View;
```

Comment:
为啥既要 export 又要 export default ？

Reply:
使用 module augmentation 的话，必须要这么使用~~~~~~

> However, there are two limitations to keep in mind:
> 1. You can’t declare new top-level declarations in the augmentation – just patches to existing declarations.
> 2. Default exports also cannot be augmented, only named exports (since you need to augment an export by its exported name, and default is a reserved word - see [#14080](https://github.com/Microsoft/TypeScript/issues/14080) for details)